www/front_src/src/Provider/index.test.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+
+import axios from 'axios';
+
+import { render, RenderResult, waitFor } from '@testing-library/react';
+import AppProvider from '.';
+import { useUserContext as mockUseUserContext } from './UserContext';
+import { UserContext } from './models';
+
+const mockedAxios = axios as jest.Mocked<typeof axios>;
+
+const retrievedUser = {
+  username: 'admin',
+  timezone: 'Europe/Paris',
+  locale: 'en-EN',
+};
+
+const retrievedActionsAcl = {
+  host: {
+    check: true,
+    acknowledgement: true,
+    downtime: true,
+  },
+  service: {
+    check: true,
+    acknowledgement: true,
+    downtime: true,
+  },
+};
+
+const retrievedTranslations = {
+  en: {
+    hello: 'Hello',
+  },
+};
+
+let userContext: UserContext | null = null;
+
+jest.mock('../App', () => {
```

Comment:
it could be interesting to extract this mock in a separate file, cause I think this will be usefull for other unit tests